### DISCUSSION TITLE
Commit message configuration suddenly partially ignored

### DISCUSSION THREAD

**Discussion by levrik** [renovatebot/renovate#32715] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Since this morning, commit messages from Renovate suddenly changed although there is an explicit configuration in place. I couldn't find any relevant change in Renovate's changelog of recent releases so I'm unsure what causes it.

It used to be `Bump @testing-library/jest-dom from 6.5.0 to 6.6.3` but it's `chore(deps): bump @testing-library/jest-dom from 6.5.0 to 6.6.3` now.

Configuration in question:

```
{
  "commitMessageAction": "Bump",
  "commitMessageTopic": "{{depName}}",
  "commitMessageExtra": "from {{currentVersion}} to {{newVersion}}"
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Read https://docs.renovatebot.com/semantic-commits/

What you're describing implies that Renovate detected some "semantic" commit prefixes from others, so decided to automatically adapt itself to those.

**Reply by levrik** [reply]
Oh my... I pushed a single commit which simply used `:` in the commit message this morning. This was rather unexpected and I would have never found out about this being the trigger by myself. Thanks for the answer!
